pytorchfi/core.py

```diff
@@ -264,7 +264,7 @@ def assert_inj_bounds(self, **kwargs):
                 and self.CORRUPT_W < self.OUTPUT_SIZE[self.CORRUPT_CONV][3]
             ), "Invalid W!"
 
-    def _set_value(self, input, output):
+    def _set_value(self, model, input, output):
```

Comment:
Why are we adding model as a parameter here? Doesn't look like its used in the function

Reply:
it's required by the PyTorch hook API. It failed otherwise.